melodic/distribution.yaml

```diff
@@ -11283,6 +11283,17 @@ repositories:
       url: https://github.com/RobotWebTools/tf2_web_republisher.git
       version: master
     status: maintained
+  tf_layer:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/igorbanfi/tf_layer-release.git
```

Comment:
Since this is such a new package, I'm going to suggest that you rename it.  `tf_layer` is very ambiguous, and actually doesn't totally describe what it does; from the description, it looks like it does costmaps from tfs.

The name used here matters, as this is the name that gets used to build the debian packages.  Imagine seeing `ros-melodic-tf-layer` in your list of packages; it doesn't tell you much about what it does, and it could be used for a lot of things.

I'll suggest renaming repository to something like `costmap_tf_layer` (GitHub will automatically redirect from the old URL), and also renaming the package to `costmap_tf_layer`.  You can choose another name, but please make it less ambiguous.

Reply:
Have renamed the package to `costmap_tf_layer`.

Reply:
Thanks for agreeing to that!  It makes it much easier to fit into the ecosystem.